cobbler/actions/mkloaders.py

```diff
@@ -128,6 +128,15 @@ def make_syslinux(self):
             self.bootloaders_dir.joinpath("menu.c32"),
             skip_existing=True,
         )
+        # According to https://wiki.syslinux.org/wiki/index.php?title=Library_modules,
+        # 'menu.c32' depends on 'libutil.c32'.
+        libutil_c32_path = self.syslinux_folder.joinpath("libutil.c32")
+        if libutil_c32_path.exists():
```

Comment:
Just saw this now. Could you please add this?

```suggestion
        if get_syslinux_version() < 5 and libutil_c32_path.exists():
```

Reply:
> Just saw this now. Could you please add this?

```python
get_syslinux_version() > 4 and libutil_c32_path.exists()
```
Is that what you mean? 'libutil.c32' exists in version '5.x+'.

Reply:
Oh yes of course, sorry. I was mixing it up with the below `if`.

Reply:
> Oh yes of course, sorry. I was mixing it up with the below `if`.

Done, please review. @SchoolGuy